src/ui/templates/components/single-radio.njk

```diff
@@ -0,0 +1,90 @@
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% import './yes-no-radio-buttons.njk' as yesNoRadioButtons %}
+{% import '../components/submit-button.njk' as submitButton %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% macro render(params) %}
+
+  {% set fieldId = params.fieldId %}
+  {% set legendText = params.legendText %}
+  {% set hintText = params.hintText %}
+  {% set hasHintHtml = params.hasHintHtml %}
+  {% set submittedAnswer = params.submittedAnswer %}
+  {% set horizontalRadios = params.horizontalRadios %}
+  {% set errorMessage = params.errorMessage %}
+  {% set conditionalYes = params.conditionalYes %}
+  {% set conditionalNo = params.conditionalNo %}
+  {% set continueButtonText = params.continueButtonText %}
+  {% set saveAndBackButtonText = params.saveAndBackButtonText %}
+  {% set saveAndBackUrl = params.saveAndBackUrl %}
+  {% set hasSaveAndBack = params.hasSaveAndBack %}
+  {% set hasCustomContent = params.hasCustomContent %}
+  {% set customContentHTML = params.customContentHTML %}
+  {% set CONTENT_STRINGS = params.CONTENT_STRINGS %}
+  {% set validationErrors = params.validationErrors %}
+  {% set submittedValues = params.submittedValues %}
+  {% set FIELDS = params.FIELDS %}
+  {% set application = params.application %}
+  {% set legendClass = params.legendClass %}
```

Comment:
Can this be ordered alphabetically and update the nunjuck imports so that our own component imports are at the bottom?